webrtc.html

```diff
@@ -11702,6 +11705,19 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
                 <td>The IdP token is invalid.</td>
               </tr>
               <tr>
+                <td><dfn><code>invalid-codecs</code></dfn></td>
+                <td>Invalid or unavailable <code>codecs[]</code>
```

Comment:
What if setParameters() attempts to modify an RTCRtpCodecParameters dictionary entry in parameters.codecs[]?  Would this be covered by InvalidModificationError ?